java/aiservices/openai/src/main/java/com/microsoft/semantickernel/aiservices/openai/chatcompletion/OpenAIChatCompletion.java

```diff
@@ -575,6 +608,12 @@ public static class Builder extends ChatCompletionService.Builder {
 
         @Override
         public OpenAIChatCompletion build() {
+            if (client == null) {
```

Comment:
This differs from OpenAITextGenerationService Builder build() method. I think it's correct there and not here. Our SK API should always throw an SKException type (which may wrap a cause).